spec/index.bs

```diff
@@ -13021,7 +13021,10 @@ a {{GPUBuffer}}, using {{GPUCommandEncoder}}.{{GPUCommandEncoder/resolveQuerySet
 
 Timestamp query requires {{GPUFeatureName/"timestamp-query"}} to be [=enabled for=] the device.
 
-Note: The timestamp values may be zero if the physical device reset timestamp counter, please ignore it and the following values.
+Note: The physical device may reset the timestamp counter occasionally, which can result in
+unexpected values such as negative deltas between timestamps that logically should be monotonically
+increasing. These instances should be rare and can safely be ignored. Applications should not be
+written in such a way that unexpected timestamps cause an application failure.
```

Comment:
Add cross-references from the two `timestampWrites` members and the `writeTimestamp` method saying something like "The timestamp written is in nanoseconds, but its value is implementation-defined and may not increase monotonically. See [[#timestamp]]."

Reply:
This is already in the [[#timestamp]] section. I've moved it to make that more apparent.

Reply:
What I meant is we should add links from timestampWrites/writeTimestamp to this section. Otherwise this text is not going to be easily found by developers, I think.

Reply:
Ah! Got it. Updated to point towards this section from the other relevant definitions.